src/scripts/Docs/Package.elm

```diff
@@ -1,42 +1,91 @@
-module Docs.Package exposing (..)
+module Docs.Package
+    exposing
+        ( Entry
+        , Metadata
+        , Module
+        , Package
+        , decode
+        , identifier
+        , remoteMetadataDecoder
+        )
 
-import Docs.Module as Module exposing (Module)
-import Docs.Version as Version exposing (Version)
+import Docs.Type exposing (Type)
+import Elm.Documentation as ElmDocs
 import Json.Decode as Decode exposing (Decoder)
-import String
-import Utils.Json
 
 
 type alias Package =
+    { metadata : Metadata
+    , modules : List Module
+    }
+
+
+type alias Metadata =
     { user : String
     , name : String
-    , version : Version
-    , modules : List Module
+    , version : String
```

Comment:
Why is this a string now? I guess that we don't need to parse into some fancy record just to turn it into a string afterwards. (This code was mostly taken from the package website where more context is needed. I had some plans to use such data at some point, but not at the moment. So everything is ok.)

Reply:
Yep, that's basically what I was thinking